### ISSUE TITLE
Popup 'notify-heading' should be an id not a class

### ISSUE THREAD

**Issue by kirsty-hames** [cgkineo/adapt-hotgrid#152]
### Subject of the issue
Same issue as Hotgraphic https://github.com/adaptlearning/adapt-contrib-hotgraphic/issues/327
